src/Tokenizer/Transformers.php

```diff
@@ -64,9 +64,11 @@ public static function create()
      */
     public function transform(Tokens $tokens)
     {
-        foreach ($tokens as $index => $token) {
-            foreach ($this->items as $transformer) {
-                $transformer->process($tokens, $token, $index);
+        foreach ($this->items as $transformer) {
+            if (\PHP_VERSION_ID >= $transformer->getRequiredPhpVersionId()) {
```

Comment:
please, move it back where it belongs.

if this check would be in `registerTransformer` - it's executed `once per transformer`
if it would be here - it's executed `once per transforer x per file` :(

Reply:
This is consistent with the fixers though, which do the same. But I'll move it back if you wish. That however will convolute the tests again.